app/assets/stylesheets/application.scss

```diff
@@ -323,3 +323,7 @@ li.group_member {
 .tooltip:hover .tooltiptext {
     visibility: visible;
 }
+
+.prediction-filter {
+	display: inline;
```

Comment:
This indenting doesn't seem consistent with the rest of the file.

Reply:
Actually it's the tooltip part that's over-indented, I'm fixing it. One thing we could look at in the future is reorganizing this massive CSS file.